admin_guide/table_management.rst

```diff
@@ -1,6 +1,51 @@
 Table Management
 $$$$$$$$$$$$$$$$$$
 
+Determining Table and Relation Size
+###################################
+
+The usual way to find table sizes in PostgreSQL, :code:`pg_total_relation_size`, drastically under-reports the size of distributed tables. All this function does on a Citus cluster is reveal the size of tables on the coordinator node. In reality the data in distributed tables lives on the worker nodes (in shards), not on the coordinator. A true measure of distributed table size is obtained as a sum of shard sizes. Citus provides helper functions to query this information.
+
++------------------------------------------+----------------------------------------------------------+
+| UDF                                      | Returns                                                  |
++==========================================+==========================================================+
+| citus_relation_size(relation_name)       | Size of actual data in table (the "main fork").          |
+|                                          |                                                          |
+|                                          | A relation can be the name of a table or an index.       |
++------------------------------------------+----------------------------------------------------------+
+| citus_table_size(relation_name)          | citus_relation_size plus                                 |
+|                                          |                                                          |
+|                                          | * size of free space map                                 |
+|                                          | * size of visibility map                                 |
++------------------------------------------+----------------------------------------------------------+
+| citus_total_relation_size(relation_name) | citus_table_size plus                                    |
+|                                          |                                                          |
+|                                          | * size of indices                                        |
++------------------------------------------+----------------------------------------------------------+
+
+Notes about all these functions:
+
+* They work only when :code:`citus.shard_replication_factor` = 1.
+* If they can't connect to a node, they error out.
+
+Here is an example of using one of the helper functions to list the sizes of all distributed tables:
+
+.. code-block:: postgresql
+
+  SELECT logicalrelid AS name,
+         pg_size_pretty(citus_table_size(logicalrelid)) AS size
+    FROM pg_dist_partition;
+
+Output:
+
+::
+
+  ┌────────────┬────────┐
+  │    name    │  size  │
+  ├────────────┼────────┤
+  │ test_table │ 256 kB │
```

Comment:
Could we show a little bit more data here? Maybe use one of our examples like github events, for which we have some data.